handler.js

```diff
@@ -350,6 +345,10 @@ async function handleJob(taskId, rcl) {
     var logDir = process.env.HF_VAR_LOG_DIR || (workDir + "/logs-hf");
     var inputDir = process.env.HF_VAR_INPUT_DIR || workDir;
     var outputDir = process.env.HF_VAR_OUTPUT_DIR || workDir;
+    
+    // make sure log directory is created
+    try { fs.mkdirSync(logDir); } catch (err) {}
+    fs.statSync(logDir);
```

Comment:
why this?

Reply:
@balis We try to make directory, even if it exists and error is thrown, we should ignore it - this is like `mkdir -p`. In next line we are making sure that directory exists.

This should be safe against race condition:
```
[client A] start
[client B] start
[client C] start
[client A] mkdir -> success
[client B mkdir -> error, ignore
[client C] mkdir -> error, ignore
[client A] stat -> success
[client B] stat -> success
[client C] stat -> success
```